server/src/prisma/config/yupValidationPrismaClient.js

```diff
@@ -0,0 +1,45 @@
+import { ValidationError } from "yup";
+
+const mutativeActions = ["create", "update", "upsert", "createMany"];
+
+class MissingSchemaError extends Error {}
+
+const customMutativeQueries = {}
+mutativeActions.forEach(action => {
+  customMutativeQueries[action] = async ({ model, operation, args, query }) => {
+    let modelSchema 
+    try {
+      const { default: schema } = await import(`./../validations/${model}Schema.js`)
+      modelSchema = schema
+    } catch (error) {
```

Comment:
we should defensively check to ensure that is in fact the error - we're errantly (see what I did there :trollface: ) assuming this is the error that is getting raised. It could be something else